x-pack/plugins/security_solution/public/timelines/components/timeline/session_tab_content/use_session_view.tsx

```diff
@@ -199,18 +201,21 @@ export const useSessionView = ({
     entityType,
     sourcererScope,
     timelineId,
-    tabType: timelineId === TimelineId.active ? TimelineTabs.session : TimelineTabs.query,
+    tabType: timelineId === TimelineId.active ? activeTab : TimelineTabs.query,
   });
 
   const sessionViewComponent = useMemo(() => {
+    const sessionViewSearchBarHeight = 118;
```

Comment:
should this be a const in session_view/common/constants? I know @opauloh was also hard coding this value in css for his fullscreen fixes. see: https://github.com/elastic/kibana/pull/129257/files#diff-9596f81cf451c350ccfa751c180aa02d876954b8d4ecba208da82a2a17245da5R26

Reply:
ideally we'd also have a test to compare the const height with the actual rendered height of the toolbar, but that can always come later :)

Reply:
ya I think we should, but probably in a different pr and remove both magic-ish numbers, just to be sure this gets in the next BC